docs/user/monitoring/monitoring-kibana.asciidoc

```diff
@@ -61,8 +61,8 @@ For more information, see {ref}/monitoring-settings.html[Monitoring settings in
 and {ref}/cluster-update-settings.html[Cluster update settings].
 --
 
-. Verify that `xpack.monitoring.enabled` and
-`xpack.monitoring.kibana.collection.enabled` are set to `true` in the 
+. Verify that `monitoring.enabled` and
+`monitoring.kibana.collection.enabled` are set to `true` in the 
 `kibana.yml` file. These are the default values. For 
```

Comment:
I see more instances of xpack.monitoring in monitoring-kibana.asciidoc.

<img width="706" alt="Screen Shot 2020-04-20 at 9 34 40 AM" src="https://user-images.githubusercontent.com/33642766/79776262-58ae5680-82ea-11ea-93ee-d2ece97db716.png">

Reply:
I only see three more in that file, and they all reference elasticsearch settings.